bundles/persistence/org.openhab.persistence.dynamodb/java/org/openhab/persistence/dynamodb/internal/DynamoDBPersistenceService.java

```diff
@@ -363,6 +371,12 @@ protected void flushBufferedData() {
             for (Entry<String, Deque<DynamoDBItem<?>>> entry : itemsByTable.entrySet()) {
                 String tableName = entry.getKey();
                 Deque<DynamoDBItem<?>> batch = entry.getValue();
+                if (dbConfig.isAutoExpirationEnabled()){
+                    for (DynamoDBItem<?> batchItem: batch){
+                        Instant expiration = batchItem.getTime().toInstant().plus(dbConfig.getAutoExpirationDays(), ChronoUnit.DAYS);
```

Comment:
`toInstant` not allowed I believe... This repo is still pre-java8?

Reply:
I'll look for a response from our maintainer friends - I poked around the various `travis.yml` scripts (odd since we use jenkins?) And all the `pom.xml` files I could find and didn't see anything obvious that limits us to features lower than 8. Could totally have skipped something though, still a new code base to me.

In any event - we have `joda` in the `lib` so I assume I can just use joda time for this? 
If you'd like me to do so, just let me know.

Reply:
As discussed in comments, java 8 is ok -- no need to change anything :+1: